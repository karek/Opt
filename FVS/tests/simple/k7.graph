a b
a c
a d
a e
a f
a g
b c
b d
b e
b f
b g
c d
c e
c f
c g
d e
d f
d g
e f
e g
f g

