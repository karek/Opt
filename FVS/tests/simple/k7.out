5
c d e f g 
